script/oracle/install-instantclient-packages.sh

```diff
@@ -45,3 +45,7 @@ rm -rf /opt/system/vendor/oracle
 ln -sf /opt/oracle/instantclient_19_6/libsqora.so.19.1 /opt/oracle/instantclient_19_6/libsqora.so
 ln -sf /opt/oracle/instantclient_19_6 /opt/oracle/instantclient
 cp config/oracle/*.ini /etc/
+
+# Fix  ORA-12637: Packet receive failed
+# https://github.com/oracle/docker-images/issues/1352
+echo "DISABLE_OOB=ON" >> /opt/oracle/instantclient/network/admin/sqlnet.ora
```

Comment:
Is it something customer also have or does this only happen on development? This file is shared with the oracle system image customers build. I do not want to set that by default if not needed

Reply:
It only happened during the development.

I see, I will update the oracle setup guide with my experience then.